2021-11-27 17:57:26,101 INFO    MainThread:3482 [wandb_setup.py:_flush():71] setting env: {}
2021-11-27 17:57:26,101 INFO    MainThread:3482 [wandb_setup.py:_flush():71] setting login settings: {}
2021-11-27 17:57:26,101 INFO    MainThread:3482 [wandb_setup.py:_flush():71] setting login settings: {}
2021-11-27 17:57:26,101 INFO    MainThread:3482 [wandb_init.py:_log_setup():371] Logging user logs to /geoinfo_vol1/home/z/h/zhao2/vit-keras/wandb/run-20211127_175726-24je73ws/logs/debug.log
2021-11-27 17:57:26,102 INFO    MainThread:3482 [wandb_init.py:_log_setup():372] Logging internal logs to /geoinfo_vol1/home/z/h/zhao2/vit-keras/wandb/run-20211127_175726-24je73ws/logs/debug-internal.log
2021-11-27 17:57:26,102 INFO    MainThread:3482 [wandb_init.py:init():404] calling init triggers
2021-11-27 17:57:26,102 INFO    MainThread:3482 [wandb_init.py:init():409] wandb.init called with sweep_config: {}
config: {}
2021-11-27 17:57:26,103 INFO    MainThread:3482 [wandb_init.py:init():449] starting backend
2021-11-27 17:57:26,103 INFO    MainThread:3482 [backend.py:_multiprocessing_setup():95] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-11-27 17:57:26,131 INFO    MainThread:3482 [backend.py:ensure_launched():199] starting backend process...
2021-11-27 17:57:26,149 INFO    MainThread:3482 [backend.py:ensure_launched():204] started backend process with pid: 3616
2021-11-27 17:57:26,151 INFO    MainThread:3482 [wandb_init.py:init():458] backend started and connected
2021-11-27 17:57:26,154 INFO    MainThread:3482 [wandb_init.py:init():519] updated telemetry
2021-11-27 17:57:26,210 INFO    MainThread:3482 [wandb_init.py:init():549] communicating current version
2021-11-27 17:57:27,416 INFO    MainThread:3482 [wandb_init.py:init():554] got version response 
2021-11-27 17:57:27,417 INFO    MainThread:3482 [wandb_init.py:init():564] communicating run to backend with 30 second timeout
2021-11-27 17:57:27,619 INFO    MainThread:3482 [wandb_init.py:init():592] starting run threads in backend
2021-11-27 17:57:32,634 INFO    MainThread:3482 [wandb_run.py:_console_start():1816] atexit reg
2021-11-27 17:57:32,636 INFO    MainThread:3482 [wandb_run.py:_redirect():1690] redirect: SettingsConsole.REDIRECT
2021-11-27 17:57:32,636 INFO    MainThread:3482 [wandb_run.py:_redirect():1695] Redirecting console.
2021-11-27 17:57:32,639 INFO    MainThread:3482 [wandb_run.py:_redirect():1751] Redirects installed.
2021-11-27 17:57:32,639 INFO    MainThread:3482 [wandb_init.py:init():619] run started, returning control to user process
2021-11-27 17:58:15,215 INFO    MainThread:3482 [wandb_run.py:_atexit_cleanup():1786] got exitcode: 255
2021-11-27 17:58:15,215 INFO    MainThread:3482 [wandb_run.py:_restore():1758] restore
